docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md

```diff
@@ -18,7 +18,7 @@ So in this case, we need to trigger four tests in sequence to achieve test the e
 
 ## Building a Transaction for This Scenario
 
-Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/environments.md)](../../../concepts/environments.md).
+Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/variable-sets.md)](../../../concepts/variable-sets.md).
```

Comment:
You have the word 'environment' still here

Reply:
Thanks.